tests/test_client.py

```diff
@@ -39,6 +39,13 @@ def test_instance(self) -> None:
 
         assert str(api) == "<Client id=microsoft-pc>"
 
+    @pytest.mark.vcr
+    def test_v1(self) -> None:
+        api = Client.open(STAC_URLS["EARTH-SEARCH"])
+        # Why, oh why, does PySTAC strip off the `stac_version` attribute?
+        # assert api.stac_version == "1.0.0"
```

Comment:
I really can't tell what this test is proving :frowning_face: so I guess I'm pro getting rid of it.

Reply:
We can open a v1.0.0 API w/o error? But yeah, its kinda silly, I'll remove.